rusoto/services/lambda/src/custom/custom_tests.rs

```diff
@@ -1,11 +1,27 @@
 extern crate rusoto_mock;
 
-use ::{Lambda, LambdaClient, InvocationRequest};
+use ::{GetPolicyRequest, GetPolicyResponse, Lambda, LambdaClient, InvocationRequest};
 
 use rusoto_core::Region;
 use rusoto_core::signature::{SignedRequest,SignedRequestPayload};
 use self::rusoto_mock::*;
 
+#[test]
+fn serialize_get_policy_response() {
+    let policy = GetPolicyResponse {
+        policy: Some("policy".into()),
+        ..GetPolicyResponse::default()
+    };
+    let mock = MockRequestDispatcher::with_status(200)
+        .with_json_body(policy.clone());
```

Comment:
previously library users would be forced to write this json out by hand. If found this error prone in practice because field names don't typically match up with struct names in letter casing and hand referencing what fields are required and which aren't is cumbersome. having the ability to use the information already encoded in the type system would be super helpful!